package.json

```diff
@@ -33,13 +33,17 @@
       "webhook": "nodes/push/webhook.js",
       "log": "nodes/push/log.js",
       "account": "nodes/config/account.js",
+      "tts": "nodes/config/tts.js",
+      "aws": "nodes/config/aws.js",
       "hangup-call": "nodes/rtcm/hangup-call.js"
     }
   },
   "files": [
     "nodes"
   ],
   "dependencies": {
+    "aws-sdk": "^2.507.0",
+    "sinon": "^7.4.2",
```

Comment:
`sinon` should be in `devDependencies` instead of `dependencies` as it's not used in production.

Reply:
@barinali moved